.github/workflows/docker.yml

```diff
@@ -1,13 +1,17 @@
-name: Docker
+name: Docker build
 
 on:
   push:
     branches-ignore:
-      - "carma-develop"
+      - "develop"
+      - "master"
+      - "release/*"
```

Comment:
it does match what carma-platform has, but could you explain to me what specifying these do in here under branches-ignore?